index.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+require_once 'functions.php';
+
+$servername = "localhost";
+$username = "root";
+$password = "password";
+$db = "vinyl-collection";
+
+$db = retrieveDb();
+
+$query = "SELECT `image`, `artist`, `album`, `year`, `genre`, `label-released` FROM `vinyl-collection`";
+
+
+$stmnt = $db->prepare($query);
+
+$stmnt->execute();
+
+$result = $stmnt->fetchAll();
+
```

Comment:
lines 21 - 28 are displaying your data, you might find it easier to vardump this information

Reply:
When I do that, it displays info in arrays.

Reply:
Cracked it, just!